### PR TITLE
remove unused form el that can be mistakenly submitted, breaking link UI

### PR DESCRIPTION
**Pull request by stuartromanek**
https://apostrophecms.atlassian.net/browse/A30U-935?atlOrigin=eyJpIjoiYmQxNWExMzBiOWFjNDkwNGE1YWNjNWI4ZGY0ZjU0OGUiLCJwIjoiaiJ9

- remove unused form. In certain states this would be submitted, causing page refresh and loss of changes

I don't know if there is value in having the form. The tiptap example goes to lengths to use a form and prevent submissions but /shrug

### CODE REVIEW

File: modules/@apostrophecms/rich-text-widget/ui/apos/tiptap-extensions/Link.js [hunk 1]
```diff
@@ -48,7 +48,9 @@ export default class Link extends Mark {
 
   commands({ type }) {
     return attrs => {
-      if (attrs.href || attrs.target || attrs.id) {
+      if (attrs.href) {
+        console.log(type);
```
Comment (boutell) [inline modules/@apostrophecms/rich-text-widget/ui/apos/tiptap-extensions/Link.js:52]: Stray log

File: modules/@apostrophecms/rich-text-widget/ui/apos/tiptap-extensions/Link.js [hunk 2]
```diff
@@ -71,6 +73,7 @@ export default class Link extends Mark {
       new Plugin({
         props: {
           handleClick(view, pos) {
+            console.log('do i ever come here');
```
Comment (boutell) [inline modules/@apostrophecms/rich-text-widget/ui/apos/tiptap-extensions/Link.js:76]: Stray log etc.

File: modules/@apostrophecms/rich-text-widget/ui/apos/tiptap-extensions/Link.js [hunk 3]
```diff
@@ -33,7 +33,7 @@ export default class Link extends Mark {
           getAttrs: dom => ({
             href: dom.getAttribute('href'),
             // Fallback for legacy name attributes
-            id: dom.getAttribute('id') || dom.getAttribute('name'),
+            // id: dom.getAttribute('id') || dom.getAttribute('name'),
```
Comment (boutell) [inline modules/@apostrophecms/rich-text-widget/ui/apos/tiptap-extensions/Link.js:36]: Can be removed?

File: modules/@apostrophecms/rich-text-widget/ui/apos/tiptap-extensions/Link.js [hunk 4]
```diff
@@ -20,11 +20,11 @@ export default class Link extends Mark {
         },
         target: {
           default: null
-        },
-        // In HTML5, the name attribute is obsolete
-        id: {
-          default: null
         }
+        // // In HTML5, the name attribute is obsolete
```
Comment (boutell) [inline modules/@apostrophecms/rich-text-widget/ui/apos/tiptap-extensions/Link.js:24]: Can be removed?

**Review by boutell — CHANGES_REQUESTED** [review]

**Comment by stuartromanek** [comment]
apologies for the noize

File: modules/@apostrophecms/rich-text-widget/ui/apos/components/AposTiptapLink.vue [hunk 5]
```diff
@@ -169,6 +167,7 @@ export default {
         delete this.value.data.target;
       }
       this.editor.commands[this.name](this.value.data);
+      console.log('set to false');
```
Comment (boutell) [inline modules/@apostrophecms/rich-text-widget/ui/apos/components/AposTiptapLink.vue:170]: Um... sorry

**Review by boutell — CHANGES_REQUESTED** [review]

**Review by boutell — APPROVED** [review]

**Pull request merged**